src/app/projects/add-edit-project/add-edit-project.component.scss

```diff
@@ -230,6 +218,69 @@ section .form-container-65 {
   z-index: 2001;
 }
 
+
+.shapefile-list {
+  width: 100%;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+
+
+  .shapefile-list-item {
+    display: flex;
+    flex-direction: column;
+    background: #96becb38;
```

Comment:
I think there might be something wrong with this colour hex.

Reply:
8 digit hex codes use the first two hex values for alpha transparency: https://caniuse.com/?search=rgba

Reply:
The coverage looks pretty good, I've always used rgba but it's good to know this exists.